config.toml

```diff
@@ -80,6 +80,10 @@ languageCode = "hi"
 languageName = "Magyar"
 languageCode = "hu"
 
+[Languages.id]
+languageName = "Indonesia"
```

Comment:
Indonesia's language name is "Indonesian" in English, or "bahasa Indonesia" in Indonesian